public/components/content-list-drawer/content-list-drawer.html

```diff
@@ -503,7 +503,7 @@
                         </li>
                         <li class="drawer__item">
                             <p class="drawer__item-title">In use by</p>
-                            <wf-presence-indicators presence-id="contentItem.composerId" in-drawer="true" ng-if="contentItem.composerId"></wf-presence-indicators>
+                            <wf-presence-indicators presence-id="contentItem.composerId" dont-display-idle="contentItem.contentType == 'liveblog'" in-drawer="true" ng-if="contentItem.composerId"></wf-presence-indicators>
```

Comment:
Would be keen to know if anyone has a better idea of what to name this new value.

It's being used to flag that live blogs should be treated differently to other content, in that it should always display the 'editing in body' Presence indicator.

Reply:
Hm, it's clear to me, although happy to hear from others 👍